stubs/slumber/slumber/exceptions.pyi

```diff
@@ -0,0 +1,13 @@
+from .utils import iterator as iterator, url_join as url_join
```

Comment:
stubgen is overzealous when creating re-exports. As these imports are not used in the stubs, they shouldn't be here. Re-exporting them is more than likely just an artifact of the implementation. Same for `serialize.pyi`.

Reply:
Thanks for sharing that! makes a lot of sense.